instrumentation/opentelemetry-instrumentation-openai/src/opentelemetry/instrumentation/openai/patch.py

```diff
@@ -0,0 +1,308 @@
+            name=span_name,
+            kind=SpanKind.CLIENT,
+            context=set_span_in_context(trace.get_current_span()),
+        )
+        _set_input_attributes(span, attributes)
+
+        try:
+            result = wrapped(*args, **kwargs)
+            if is_streaming(kwargs):
+                return StreamWrapper(
+                    result,
+                    span,
+                    function_call=kwargs.get("functions") is not None,
+                    tool_calls=kwargs.get("tools") is not None,
+                )
+            else:
+                _set_response_attributes(span, result)
+                span.end()
+                return result
+
+        except Exception as error:
+            span.set_status(Status(StatusCode.ERROR, str(error)))
+            span.set_attribute("error.type", error.__class__.__name__)
+            span.end()
+            raise
+
+    return traced_method
+
+
+@silently_fail
+def _set_input_attributes(span, attributes: LLMSpanAttributes):
+    for field, value in attributes.model_dump(by_alias=True).items():
+        set_span_attribute(span, field, value)
+
+
+@silently_fail
+def _set_response_attributes(span, result):
+    set_span_attribute(
+        span, GenAIAttributes.GEN_AI_RESPONSE_MODEL, result.model
+    )
+    print(result)
+    if getattr(result, "choices", None):
+        choices = result.choices
+        responses = [
+            {
+                "role": (
+                    choice.message.role
+                    if choice.message and choice.message.role
+                    else "assistant"
+                ),
+                "content": extract_content(choice),
+                **(
+                    {
+                        "content_filter_results": choice[
+                            "content_filter_results"
+                        ]
+                    }
+                    if "content_filter_results" in choice
+                    else {}
+                ),
+            }
+            for choice in choices
+        ]
+        for choice in choices:
+            if choice.finish_reason:
+                set_span_attribute(
+                    span,
+                    GenAIAttributes.GEN_AI_RESPONSE_FINISH_REASONS,
+                    choice.finish_reason,
+                )
+        set_event_completion(span, responses)
+
+    if getattr(result, "id", None):
+        set_span_attribute(span, GenAIAttributes.GEN_AI_RESPONSE_ID, result.id)
+
+    # Get the usage
+    if getattr(result, "usage", None):
+        set_span_attribute(
+            span,
+            GenAIAttributes.GEN_AI_USAGE_INPUT_TOKENS,
+            result.usage.prompt_tokens,
+        )
+        set_span_attribute(
+            span,
+            GenAIAttributes.GEN_AI_USAGE_OUTPUT_TOKENS,
+            result.usage.completion_tokens,
+        )
+        set_span_attribute(
+            span,
+            "gen_ai.usage.total_tokens",
+            result.usage.total_tokens,
```

Comment:
it's not in the semantic conventions, the primary reason is that it's redundant, so the suggesting is to remove it
```suggestion
```

Reply:
agree will remove